src/components/DialogBox/HexBinDec.vue

```diff
@@ -1,136 +1,134 @@
 <template>
-    <v-dialog
-        v-model="SimulatorState.dialogBox.hex_bin_dec_converter_dialog"
-        :persistent="false"
-    >
-        <v-card class="messageBoxContent">
-            <v-card-text>
-                <p class="dialogHeader">Hex-Bin-Dec Convertor</p>
-                <v-btn
-                    size="x-small"
-                    icon
-                    class="dialogClose"
-                    @click="
-                        SimulatorState.dialogBox.hex_bin_dec_converter_dialog = false
-                    "
-                >
-                    <v-icon>mdi-close</v-icon>
-                </v-btn>
-                <div
-                    v-for="inputEle in inputArr"
-                    id="bitconverterprompt"
-                    :key="inputEle.inputId"
-                    title="Dec-Bin-Hex-Converter"
-                >
-                    <label>{{ inputEle.label }}</label>
-                    <br />
-                    <input
-                        :id="inputEle.inputId"
-                        type="text"
-                        :value="inputEle.val"
-                        :label="inputEle.label"
-                        name="text1"
-                        @keyup="() => converter(inputEle.inputId)"
-                    />
-                    <br /><br />
-                </div>
-            </v-card-text>
-            <v-card-actions>
-                <v-btn class="messageBtn" block @click="setBaseValues(0)">
-                    Reset
-                </v-btn>
-            </v-card-actions>
-        </v-card>
-    </v-dialog>
+  <v-dialog v-model="SimulatorState.dialogBox.hex_bin_dec_converter_dialog" :persistent="false">
```

Comment:
can you please use the current format of template with prettier instead of using volar format ?
as it is hard to review changes